### ISSUE TITLE
Fix duplication of side navigation

### ISSUE THREAD

**Issue by atsansone** [flutter/website#9708]
a11y script found duplicate left navigation data. This code exists in [lines 38-40](https://github.com/flutter/website/blob/main/src/_includes/header.html#L38-L40) header.html and [line 17](https://github.com/flutter/website/blob/main/src/_layouts/default.html#L17) of default.html. This raises multiple [WCAG F77 errors](https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/#sc4_1_1).  This causes issues with accessibility technology. These lines should be hidden.

```html
          <div class="site-sidebar site-sidebar--header d-md-none">
            {% include_cached sidenav-level-1.html nav=site.data.sidenav page_url_path=route %}
          </div>
```
